scripts/entrypoint.sh

```
@@ -0,0 +1,91 @@
+seed_volume "Bookmarks" "/app/.initial-bookmarks" "/app/data/bookmarks"
+if [ $? -ne 0 ]; then
+  echo "[Entrypoint] CRITICAL ERROR: Seeding Bookmarks failed due to a copy error."
+  CRITICAL_SEEDING_FAILED=1
+fi
+
+if [ $CRITICAL_SEEDING_FAILED -ne 0 ]; then
+  echo "[Entrypoint] One or more critical data seeding operations experienced a copy failure. Halting container."
+  exit 1
+fi
+
+echo "[Entrypoint] Volume seeding process complete (or skipped for non-empty volumes / non-critical failures). Executing CMD: $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)